### PR TITLE
Add a new article

### PR DESCRIPTION
**Pull request by enisn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,8 @@ Add GitHub topic [`csharp-sourcegenerator`](https://github.com/topics/csharp-sou
...
 - [SourceGeneratorPlayground](https://wengier.com/SourceGeneratorPlayground) - an online Source Generator Playground to play with generator ideas 💡 without any setup noise. [Source repo](https://github.com/davidwengier/SourceGeneratorPlayground).
 - [davidwengier/SourceGeneratorTemplate](https://github.com/davidwengier/SourceGeneratorTemplate)  - ![stars](https://img.shields.io/github/stars/davidwengier/SourceGeneratorTemplate?style=flat-square&cacheSeconds=604800) ![last commit](https://img.shields.io/github/last-commit/davidwengier/SourceGeneratorTemplate?style=flat-square&cacheSeconds=86400) A basic template for writing a C# source generator, from the Roslyn dev.
-
+- [Mastering at Source Generators](https://medium.com/c-sharp-progarmming/mastering-at-source-generators-18125a5f3fca) Sample about creating csharp files from an entity using different templates such as Controller, Domain Service, Repository.
+-
```
Comment (amis92) [inline README.md:18]: Please put it in the Articles section (next to last) and add `(date)` as other items have.

**Review by amis92 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (enisn) [reply README.md:18]: Sorry, my bad, I couldn't catch the section

Comment (enisn) [reply README.md]: Can you review again @amis92

**Review by amis92 — APPROVED** [review]
Thanks!

**Comment by amis92** [comment]
@enisn I've read that article, but to be honest - your `TodoController` template could've been achieved (in this form) by creating a generic `CrudControllerBase<TDto>` and using `TDto` instead of `{{ClassName}}` everywhere (except constructor name ;) ). I don't see the value added from using Source Generators to do this, quite the opposite - since you're using a `.txt` file there's no syntax support/intellisense available when editing it. 🤷‍♂️

**Pull request merged**